phpBB/phpbb/db/migration/data/v33x/bot_update_vol2.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v33x;
+
+class bot_update_vol2 extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return ['\phpbb\db\migration\data\v33x\v334'];
+	}
+
+	public function update_data()
+	{
+		return [
+			['custom', [[$this, 'add_bots']]],
+		];
+	}
+
+	public function add_bots()
+	{
+		$bots = [
+			'Ahrefs [Bot]' => 'AhrefsBot/',
+			'Amazon [Bot]' => 'Amazonbot/',
+			'Semrush [Bot]' => 'SemrushBot/'
```

Comment:
I see the other arrays have a trailing comma for the last item as well. Shall I add this, given the tests do not fail on it anyway?

Reply:
I do not think is necessary, probably though by the coding guidelines... I do not recall now.

Reply:
Please add it.